pyproject.toml

```diff
@@ -26,7 +26,7 @@ dynamic = [
   "version",
 ]
 dependencies = [
-  "aiohttp>=3.10.2,<3.11",
+  "aiohttp==3.11.14",
```

Comment:
Currently, Test CI doesn't run automatically somehow. We can trigger test manually from Actions. And the test was failed when we upgrade the `aiohttp` version.
https://github.com/tier4/otaclient-iot-logging-server/actions/runs/14104642816

Because `aiohttp` stops to support python3.8 from `3.11.x`.
https://docs.aiohttp.org/en/stable/changes.html#id38

It means we can't upgrade `aiohttp` as long as we support python3.8.

Reply:
Seems `3.10.11` is available and fixed the vulnerability. How about to pin at `3.10.11`?